src/crafting.h

```diff
@@ -49,7 +49,10 @@ struct recipe {
         requirement_data requirements;
         std::vector<byproduct> byproducts;
         std::string cat;
-        bool contained; // Does the item spawn contained?
+        // Does the item spawn contained in container?
```

Comment:
Currently all are specified in the implementation.
The crafting system will need a more thorough rework, but not in this PR.